java/src/apps/startup/configurexml/StartupActionsPreferencesPanelXml.java

```diff
@@ -2,9 +2,11 @@
 
 import apps.StartupActionsManager;
 import apps.startup.StartupModel;
+import jmri.ConfigureManager;
 import jmri.InstanceManager;
 import jmri.configurexml.AbstractXmlAdapter;
 import org.jdom2.Element;
+import org.python.jline.internal.Log;
```

Comment:
> We've been using the {} construct for a long time. 3484 matches as I see it. Doing that, and letting the logger defer the string construction, was one of the reasons for not needing as many if debug constructs.

All true. But note we did all that _without_ any org.python imports. 

I think your IDE is confused about something it sees, and is incorrectly deciding to include that import. 

The right way to fix this is to first understand what settings/options are causing the IDE to think that.